doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -148,7 +148,9 @@ nRF Machine Learning (Edge Impulse)
 nRF Desktop
 -----------
 
-|no_changes_yet_note|
+* Added the ``nrfdesktop-wheel-qdec`` DT alias support to :ref:`nrf_desktop_wheel`.
+  The alias must be used to specify QDEC instance used for scroll wheel if used board supports multiple QDEC instances (for example ``nrf54l15pdk_nrf54l15_cpuapp``).
+  The alias does not need to be defined if used board supports only one QDEC instance, because in that case wheel module can rely on ``qdec`` DT label provided by the board.
```

Comment:
```suggestion
  You do not need to define the alias if your board supports only one QDEC instance, because in that case, the wheel module can rely on the ``qdec`` DT label provided by the board.
```